src/main/java/net/sf/jabref/wizard/text/gui/TextInputDialog.java

```diff
@@ -479,9 +482,13 @@ private void updateSourceView() {
     }
 
     private String[] getAllFields() {
-        List<String> texFields = EntryTypes.getType(entry.getType(), frame.getCurrentBasePanel().getBibDatabaseContext().getMode()).getAllFields();
+        List<String> texFields = new ArrayList<>();
+        Optional<EntryType> type = EntryTypes.getType(entry.getType(),
+                frame.getCurrentBasePanel().getBibDatabaseContext().getMode());
+        if (type.isPresent()) {
+            texFields = type.get().getAllFields();
```

Comment:
You will need to do `texFields.addAll(type.get().getAllFields());` as the List returned from getAllFields can not be edited. I fixed this bug in the old version in #760 (which will collide with this PR...).